browser/src/react-redux/App/LoginPage.jsx

```diff
@@ -0,0 +1,91 @@
+import React from 'react'
```

Comment:
Facebook's create-react-app now warns when you use JSX. I like to follow their conventions. Let's stick to `.js` files only now. Go ahead and change this extension.